fenix/app/src/main/java/org/mozilla/fenix/library/LibrarySiteItemView.kt

```diff
@@ -99,4 +104,8 @@ class LibrarySiteItemView @JvmOverloads constructor(
     enum class ItemType {
         SITE, FOLDER
     }
+
+    companion object {
+        private const val OVERFLOW_EXTRA_DIPS = 16
```

Comment:
I see this View already having 40dp height/width - [the favicon](https://searchfox.org/mozilla-mobile/rev/2539bc45c927cd8ec585928216685141b6e7ec25/firefox-android/fenix/app/src/main/res/layout/library_site_item.xml#14-29) uses [history_favicon_width_height](https://searchfox.org/mozilla-mobile/rev/2539bc45c927cd8ec585928216685141b6e7ec25/firefox-android/fenix/app/src/main/res/values/dimens.xml#16).
So we'd only need to increase the tap area with (48 - 40) / 2 => 4dp.
(Want to also note again that we need this only for the favicon, not for the overflow icon)